lib/strategies/base.js

```diff
@@ -80,6 +80,10 @@ BaseStrategy.prototype._reformatInput = function () {
 
   input.value = formattedState.value;
   setSelection(input, formattedState.selection.start, formattedState.selection.end);
+
+  if (this._afterReformatInput) {
```

Comment:
What are your thoughts on implementing `_afterReformatInput` in `BaseStrategy` and having it be abstract-ish? That would prevent BaseStrategy from having code specifically for AndroidChromeStrategy.